addon-test-support/helpers.js

```diff
@@ -53,9 +53,9 @@ function fireEvent(element, type, options = {}) {
   element.dispatchEvent(event);
 }
 
-function buildBasicEvent(type, options = {}) {
+function buildBasicEvent(type, options = {}, bubbles = true, cancelable = true) {
   let event = document.createEvent('Events');
-  event.initEvent(type, true, true);
+  event.initEvent(type, bubbles, cancelable);
```

Comment:
Needed a way to fire a focus event when the document (window) doesn't have focus. The focus event is does not 'bubble'